distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso

```diff
@@ -92,6 +94,59 @@ read path=(Missing_Argument.throw "path") format=Auto_Detect (on_problems : Prob
         if file_obj.is_directory then Error.throw (Illegal_Argument.Error "Cannot `read` a directory, use `Data.list`.") else
             file_obj.read format on_problems
 
+## ALIAS load, open
+   GROUP Input
+   ICON data_input
+   Reads a a list of files into Enso.
+
+   Arguments:
+   - paths: A list of files to load. It can be a Vector, Column or Table of
+     files, paths or URIs to fetch. If a Table is provided, it must either
+     contain a single column or a column called `path` (case insensitive).
+   - format: A `File_Format` object used to read files into memory.
+     If `Auto_Detect` is specified; each file determines the specific
+     type and configures it appropriately. If there is no matching type then
+     a `File_Error.Unsupported_Type` error is returned.
+   - return: Specifies the shape of the data to return.
```

Comment:
I'm not sure that I understand what a user would do with this option. Should it be on the graphical interface API?

Reply:
OK. I get what this is now. Guess we will improve the comment as we add more options.

Reply:
Happy to accept suggestions on what the doc could be. Perhaps I should add examples?

I hope in GUI it will be easy because we'll have a dropdown